addons/tm_tmf_loadouts/CfgLoadouts.hpp

```diff
@@ -335,7 +359,24 @@ class CfgLoadouts
 		category = "Armaverse Factions";
 		#include "loadouts\sla_1990.hpp"
 	};
-	
+	class HIDF_1990
+	{
+		displayName = "Horizon Island Defence Force (1990)";
+		category = "Armaverse Factions";
+		#include "loadouts\hidf_1990.hpp"
+	};
+	class HIDF_2000
+	{
+		displayName = "Horizon Island Defence Force (2000)";
+		category = "Armaverse Factions";
+		#include "loadouts\hidf_2000.hpp"
+	};
+	class SYNDIKAT_1990
+	{
+		displayName = "Syndikat (1990-, Bandits)";
```

Comment:
typo

Reply:
Not a typo, they're called Syndikat with a K.

Reply:
i meant the - after 1990, shoudl have clarified, sorry. Also what does Bandits indicate?

Reply:
the - indicates from 1990 and forwards. As for Bandits, the vanilla faction is split up between paramilitary and bandits.